README.md

```diff
@@ -31,7 +31,7 @@ In this example, the Ballerina Kafka Connector is used to connect Ballerina to A
 - [Apache Kafka 1.1.0](https://kafka.apache.org/downloads)
   * Download the binary distribution and extract the contents
 - [Ballerina Kafka Connector](https://github.com/wso2-ballerina/package-kafka)
-  * After downloading and extracting, build it using maven and copy its .jar files into the `<BALLERINA_HOME>/bre/lib` folder
+  * Clone or download the source, build it using maven  and extract the distribution created at `/component/target/wso2-kafka-<version>.zip`. Run the install.{sh/bat} script to install the package.
```

Comment:
I think this is a bit misleading. We have already hosted the build distribution in [here](https://github.com/wso2-ballerina/package-kafka/releases) . It contains the install scripts. So we don't need to build the source again. We can directly use the distribution or if the user want then can build the source and continue.